.stack-to-nix.cache

```diff
@@ -0,0 +1,20 @@
+https://github.com/input-output-hk/cardano-ledger fbffd18089041efb1f8d1896c5410bc2302811ef . 1h15aiq09ry1xmf6zwl2kwg4j913fzh42qfamp0807381sgfpaxd cardano-ledger cardano-ledger.nix
```

Comment:
Best not to check this file in. It causes problems (input-output-hk/haskell.nix#57).